src/stream/src/executor/over_window/over_partition.rs

```diff
@@ -248,12 +251,28 @@ impl<'a, S: StateStore> OverPartition<'a, S> {
         calls: &'a [WindowFuncCall],
         row_conv: RowConverter<'a>,
     ) -> Self {
+        let rows_frames = calls
+            .iter()
+            .filter_map(|call| call.frame.bounds.as_rows())
+            .collect::<Vec<_>>();
+        // TODO(rc): maybe should avoid repeated merging
```

Comment:
Is this comment still relevant? Seems like we only merge them once here.

Reply:
`OverPartition` is a temporary structure for frame finding and cache maintanence, so it will be constructed once per chunk.